qa_eval/steps/__init__.py

```diff
@@ -95,9 +104,11 @@ def get_steps_matches(
 
 def evaluate_steps(
     reference_steps_groups: list[list[dict]],
-    actual_steps: list[dict]
+    actual_steps: list[dict],
+    matches : list[tuple[int, int, int, float]] | None = None
```

Comment:
The code is poorly formatted

Reply:
Meaning?

In a refactoring PR, I want to define a type `Match`, which would shorten function definitions.

Reply:
Meaning there is a white space before the colon

Reply:
Fixed